cmd/ipfs/daemon.go

```diff
@@ -136,6 +137,7 @@ future version, along with this notice. Please move to setting the HTTP Headers.
 		cmds.BoolOption(unencryptTransportKwd, "Disable transport encryption (for debugging protocols)").Default(false),
 		cmds.BoolOption(enableGCKwd, "Enable automatic periodic repo garbage collection").Default(false),
 		cmds.BoolOption(adjustFDLimitKwd, "Check and raise file descriptor limits if needed").Default(false),
+		cmds.BoolOption(offlineKwd, "Run offline. Do not connect with rest of the network but provide local API.").Default(false),
```

Comment:
"Run offline. Do not connect to the rest of the network, but provide only the local API." 👍